packages/x-components/src/x-modules/search/components/sort-picker-list.vue

```diff
@@ -12,7 +12,7 @@
       :class="[cssClasses, buttonClass]"
       data-test="sort-picker-button"
       :events="event"
-      :aria-pressed="item === selectedSort"
+      :aria-pressed="item === selectedSort || null"
```

Comment:
https://v3-migration.vuejs.org/breaking-changes/attribute-coercion.html

Reply:
Vue3 will remove `aria-pressed` attr if the value is `null`. https://v3-migration.vuejs.org/breaking-changes/attribute-coercion.html#comparison-between-2-x-3-x-behavior

With the aim of leaving as it is, I would use `toString()` to keep `aria-pressed="false"`

![Screenshot 2024-05-01 at 2 26 44 PM](https://github.com/empathyco/x/assets/10746604/9fb34b21-a60b-4bb9-85c4-f7f840bf1ed2)
VS
![Screenshot 2024-05-01 at 2 27 12 PM](https://github.com/empathyco/x/assets/10746604/84d7787d-24c4-4609-a27b-414cd81661b2)